src/o365/graph/commands/schemaextension/schemaextension-get.spec.ts

```diff
@@ -0,0 +1,326 @@
+    auth.service.resource = 'https://graph.microsoft.com';
+    cmdInstance.action = command.action();
+    cmdInstance.action({
+      options: {
+        debug: false,
+        id: 'ext6kguklm2_TestSchemaExtension',
+      }
+    }, () => {
+      try {
+        assert(cmdInstanceLogSpy.calledWith({
+          "@odata.context": "https://graph.microsoft.com/v1.0/$metadata#schemaExtensions/$entity",
+          "id": "ext6kguklm2_TestSchemaExtension",
+          "description": "Test Description",
+          "targetTypes": [
+            "Group"
+          ],
+          "status": "InDevelopment",
+          "owner": "b07a45b3-f7b7-489b-9269-da6f3f93dff0",
+          "properties": [
+            {
+              "name": "MyInt",
+              "type": "Integer"
+            },
+            {
+              "name": "MyString",
+              "type": "String"
+            }
+          ]
+        }));
+        done();
+      }
+      catch (e) {
+        done(e);
+      }
+    });
+  });
+
+  it('gets schema extension (debug)', (done) => {
+    sinon.stub(request, 'get').callsFake((opts) => {
+      if (opts.url.indexOf(`https://graph.microsoft.com/v1.0/schemaExtensions`) === 0) {
+        return Promise.resolve({
+          "@odata.context": "https://graph.microsoft.com/v1.0/$metadata#schemaExtensions/$entity",
+          "id": "ext6kguklm2_TestSchemaExtension",
+          "description": "Test Description",
+          "targetTypes": [
+            "Group"
+          ],
+          "status": "InDevelopment",
+          "owner": "b07a45b3-f7b7-489b-9269-da6f3f93dff0",
+          "properties": [
+            {
+              "name": "MyInt",
+              "type": "Integer"
+            },
+            {
+              "name": "MyString",
+              "type": "String"
+            }
+          ]
+        });
+      }
+
+      return Promise.reject('Invalid request');
+    });
+
+    auth.service = new Service();
+    auth.service.connected = true;
+    auth.service.resource = 'https://graph.microsoft.com';
+    cmdInstance.action = command.action();
+    cmdInstance.action({
+      options: {
+        debug: true,
+        id: 'ext6kguklm2_TestSchemaExtension',
+      }
+    }, () => {
+      try {
+        assert(cmdInstanceLogSpy.calledWith({
+          "@odata.context": "https://graph.microsoft.com/v1.0/$metadata#schemaExtensions/$entity",
+          "id": "ext6kguklm2_TestSchemaExtension",
+          "description": "Test Description",
+          "targetTypes": [
+            "Group"
+          ],
+          "status": "InDevelopment",
+          "owner": "b07a45b3-f7b7-489b-9269-da6f3f93dff0",
+          "properties": [
+            {
+              "name": "MyInt",
+              "type": "Integer"
+            },
+            {
+              "name": "MyString",
+              "type": "String"
+            }
+          ]
+        }));
+        done();
+      }
+      catch (e) {
+        done(e);
+      }
+    });
+  });
+
+
```

Comment:
Could you please add an extra test to test `/schemaExtensions` endpoint in a case of failure?

For example schema not found response.
Failure response for schema not found is with status code 404:
```
{
    "error": {
        "code": "ResourceNotFound",
        "message": "Cannot find resource with id: 123.",
        "innerError": {
            "request-id": "4827c0cb-3580-4e93-98db-cbac18e9951f",
            "date": "2019-03-25T14:09:57"
        }
    }
}
```

Reply:
Same here, the path should be `schemaextension` not `schmeaextension`